scripts/01_Seurat_QC.Rmd

```diff
@@ -1,115 +1,103 @@
 ---
 title: "Seurat QC and Object for `r params$sample_name`"
-author: 'User ID: `r Sys.getenv("USER")`'
-date: "`r Sys.Date()`"
+author: 'Antonia Chroni <chronia@chop.edu> for D3B'
+date: "2023"
 output:
-  html_document:
-    fig_height: 7
-    fig_width: 9
-    keep_md: yes
-    toc: yes
-    df_print: paged
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
 params:
-  out_path: './' # path to dir to save all output files
-  data_path: '/path/to/10x/output/dir/filtered_feature_bc_matrix/' # path to the directory with the filtered_feature_bc_matrix
-  sample_name: "sample_name"   # name of the current sample
-  samples: sample_name
-  scooter_path: './'
-  hashtag_ID: NULL # names of the hashtags to keep (if the hashtags are NOT separate from the gene expression data)
-  hashtags_to_keep: NULL # names of the hashtags to keep (if the hashtags are NOT separate from the gene expression data)
-  hashtag_file: NULL # path to hashtag file (if it is separate from the gene expression data)
-  assay: "RNA"   # Assay we are working with
+  results_path: './' # path to dir to save all output files
+  data_path: '/path/to/10x/output/dir/filtered_feature_bc_matrix/'
+  scooter_path: './' # File path to scooter directory
+  sample_name: "pbmc_1k_v3"   # name of the sample in the library
+  assay: "RNA"   # Assay 
   grouping: "orig.ident" # grouping for plots
-  hash_grouping: "hash.ID" # grouping for plots hashtags but doesnt work with anythin else right now
-  min_genes: 400 # minimum number of genes per cell
-  max_genes: 4000 # maximum number of genes per cell
-  max_mt: 5  # maximum percent mitochondrial reads per cell
+  min_genes: 200 # minimum number of genes per cell
+  max_genes: 6000 # maximum number of genes per cell
+  max_mt: 15  # maximum percent mitochondrial reads per cell
   normalize_method: "log_norm" # normalization method. One of log_norm or sct
-  nfeatures: 2000 # number of variable features to extract
```

Comment:
To confirm, this should now be hardcoded and no longer an option a user provides? It's ok if it is, just checking for in/when we modify the tool to work with this update